src/site/content/en/blog/payment-and-address-form-best-practices/index.md

```diff
@@ -0,0 +1,782 @@
+
+{% Aside %}
+The [datalist](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/datalist) element enables 
+a user to select from a list of available options and provides matching suggestions as the user enters text. 
+Try out datalist for `text`, `range` and `color` inputs at [simpl.info/datalist](https://simpl.info/datalist).
+For birth year input, you can compare a select with an input and datalist at [datalist-select.glitch.me](https://datalist-select.glitch.me).
+{% endAside %}
+
+#### Use autocomplete to improve accessibility and help users avoid re-entering data {: #autocomplete-attribute}
+
+Using appropriate `autocomplete` values enables browsers to help users by securely storing data and 
+autofilling `input`, `select` and `textarea` values. This is particularly important on mobile, and 
+crucial for avoiding [high form abandonment rates](https://www.zuko.io/blog/8-surprising-insights-from-zukos-benchmarking-data). Autocomplete also provides [multiple accessibility benefits](https://www.w3.org/WAI/WCAG21/Understanding/identify-input-purpose.html).
+
+If an appropriate autocomplete value is available for a form field, you should use it. [MDN web docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete) has a full list of values and explanations of how to use 
+them correctly.
+
+{: #stable-name-id}
+
+{% Aside %}
+As well as using appropriate autocomplete values, help browsers autofill forms by giving form fields 
+`name` and `id` attributes [stable values](#stable-name-id) that don't change between page loads or 
+website deployments.
+{% endAside %}
+
+{: #billing-address}
+
+By default, set the billing address to be the same as the delivery address. Reduce visual clutter by 
+providing a link to edit the billing address (or use [summary and details elements](https://simpl.info/details/)) 
+rather than displaying the billing address in a form.
+
+<figure class="w-figure">
+  <img src="images/review-order.png" alt="Example checkout page showing link 
+  to change billing address.">
+  <figcaption class="w-figcaption">Add a link to review billing.</figcaption>
+</figure>
+
+Use appropriate autocomplete values for billing name and address, just as you do for shipping address, 
+so the user doesn't have to enter the same data twice. Add a prefix word to autocomplete attributes 
+if you have different values for inputs with the same name in different sections.
+
+```html
+<input autocomplete="shipping address-line-1" ...>
+...
+<input autocomplete="billing address-line-1" ...>
+```
+
+#### Help users enter the right data {: #validation}
+
+Try to avoid 'telling off' customers because they 'did something wrong'. Instead, help users complete 
+forms more quickly and easily by helping them fix problems as they happen. Through the checkout process, 
+customers are trying to give your company money for a product or service—your job is to assist them, 
+not to punish them!
```

Comment:
```suggestion
Try to avoid "telling off" customers because they "did something wrong". Instead, help users complete 
forms more quickly and easily by helping them fix problems as they happen. Through the checkout process, 
customers are trying to give your company money for a product or service—your job is to assist them, 
not to punish them!
```